pixl_core/tests/test_upload.py

```diff
@@ -0,0 +1,81 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Test functionality to upload files to an endpoint."""
+
+
+from datetime import datetime, timezone
+
+import pytest
+from core._database import get_project_slug_from_db, update_exported_at_and_save
+from core.database import Image
+from core.upload import upload_content, upload_dicom_image
+
+
+@pytest.mark.usefixtures("run_containers")
+def test_upload_content(data, mounted_data) -> None:
+    """Tests that file is present on the endpoint after upload"""
+    local_file = data / "public.zip"
+    with local_file.open("rb") as handle:
+        upload_content(handle, remote_file="public.zip", remote_dir="new_dir")
+
+    assert (mounted_data / "new_dir" / "public.zip").exists()
```

Comment:
probably can remove this test as we've got that covered in the upload dicom image?